Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -957,10 +966,40 @@ def get_incidents_command(client, args):
     )
 
 
+def check_if_incident_was_modified_in_xdr(incident_id):
+    demisto_incident = demisto.get_incidents()[0]
+    last_mirrored_in_time = demisto_incident.get('CustomFields', {}).get('lastmirroredintime')
+    last_mirrored_in_time_timestamp = arg_to_timestamp(last_mirrored_in_time, 'last_mirrored_in_time')
+
+    last_modified_incidents_dict = get_integration_context().get('modified_incidents', {})
```

Comment:
pass the modified_incidents to the function.
Avoid using globals like `get_integration_context()`